### ISSUE TITLE
when cookbook breaks on an element, include that element's phil-granted identifier

### ISSUE THREAD

**Issue by kswanson33** [openstax/cnx-recipes#5092]
### User Story

As a content manager, I want to know which CNXML element is making cookbook blow up so I can fix it.

### Context

A new identification attribute has been added as of openstax/enki#216. 

> Note: It only adds the attribute when running locally (when the LOCAL_ATTIC_DIR environment variable is set)

### Acceptance Criteria

- [ ] When cookbook blows up, it logs the `data-sm` attribute (if available)

**Comment by kswanson33** [comment]
@phil for when you're back:

> I'm interested in feedback on the value of the attribute. Currently it's ./modules/m123/index.cnxml:123:456 where 123 is the line and 456 is the column. I'm hpoing that folks will not parse the string though

Re: folks not parsing the string -- is it desired that only module number, column, and line will be logged? In that case, for the value of the attribute, something like "M123:C456:L789" would be most helpful. However, the current attribute with the full path would let CMs click it to open the file (i think?), which might be more desirable.

**Comment by philschatz** [comment]
Yeah, click-in-a-VSCode-terminal was totally the inspiration for the initial format!

Since VSCode detects it, there's probably an easy way to extract these lines from a log file which would allow CORGI to extract them and turn them into links (e.g. a GitHub [link](https://github.com/openstax/osbooks-american-government/blob/main/modules/m90429/index.cnxml#L90C182) or GitPod/POET [link](https://gitpod.io/#https://github.com/openstax/osbooks-american-government/blob/main/modules/m90429/index.cnxml))

I've also been using the full path to the module because I'm hoping to loosen the requirement that all modules be `/modules/m#####/index.cnxml` and allow enki to not really care about the path to the CNXML files (e.g. `/Unit 1/Ch 4/Kinematics.cnxml` or `/content/04-kinematics.cnxml`)

**Comment by kswanson33** [comment]
logging format: `M123:L456:C789`

**Comment by Stackblocks** [comment]
For Tuesday: 

@lwtchu we need to break something other than the intro class because that has a bespoke error message already.

**Comment by LostMojo** [comment]
In order to get data-mapping done on enki, be sure to set `LOCAL_ATTIC_DIR=1` when running enki-- for example, the command you might put into your terminal might look like `LOCAL_ATTIC_DIR=1 ./enki --data-dir ./data/organic-chemistry --command all-pdf --repo openstax/osbooks-organic-chemistry --book-slug organic-chemistry --ref main`
